src/mono/wasm/runtime/es6/dotnet.es6.lib.js

```diff
@@ -7,8 +7,10 @@
 
 // because we can't pass custom define symbols to acorn optimizer, we use environment variables to pass other build options
 const DISABLE_LEGACY_JS_INTEROP = process.env.DISABLE_LEGACY_JS_INTEROP === "1";
+const ENABLE_BROWSER_PROFILER = process.env.ENABLE_BROWSER_PROFILER === "1";
+const ENABLE_AOT_PROFILER = process.env.ENABLE_AOT_PROFILER === "1";
 
-function setup(disableLegacyJsInterop) {
+function setup(linkerDisableLegacyJsInterop, linkerEnableAotProfiler, linkerEnableBrowserProfiler) {
```

Comment:
is there a way to make these not positional arguments, and a dict with named members instead?